README.md

```diff
@@ -49,6 +49,17 @@ Busola is delivered with the following default settings:
 | `navigation.externalNodes` | A nested list of links to external websites. `category`, `icon`: a category name and optional icon / `children`: a list of pairs (label and link) | category: `Learn more` / children: `Kyma Documentation`, `Our Slack`, `Github`                                                                                                                          |
 | `version`                  | Configuration version. Don’t edit this. Can be empty.                                                                                             | the most recent release                                                                                                                                                                                 |
 
+### Configuration sources
+
+By default the configuration is gathered from four sources:
```

Comment:
If you say "by default" - is it possible for the reader to change that default, like adding/removing sources or changing the priority? Or is it not just the default way, but the only way at all?

I wouldn't mention the number, because if it ever changes, you need to remember overwriting the number too :D 

```suggestion
Busola configuration is the product of gathering and merging the configurations from several individual sources. The following list presents the sources in the order of precedence, with init params having the highest priority over the others:```

Reply:
Daaaamn... I was thinking about 1) mentioning the number (yes, it is another place to change!) and 2) meaning of "by default". Live and learn I guess.